test/e2e/app-dir/ppr-metadata-streaming/next.config.js

```diff
@@ -1,11 +1,6 @@
 /**
  * @type {import('next').NextConfig}
  */
-const nextConfig = {
-  experimental: {
-    ppr: true,
```

Comment:
this test is called `ppr-metadata-streaming` so it should still have PPR enabled